tests/testenv-container/Dockerfile

```diff
@@ -67,4 +70,4 @@ COPY krb5.conf /krb5.conf
 COPY config.ini /etc/config.ini
 
 # Code file to execute when the docker container starts up (`make-test.sh`)
-ENTRYPOINT ["/make-test.sh"]
+ENTRYPOINT [PIPELINE_TYPE=$PIPELINE_TYPE /make-test.sh]
```

Comment:
Dockerfile has `ENV` directive. Alternatively there is an `--env` option to compose to set the environment variable.
- https://docs.docker.com/reference/dockerfile/#env
- https://docs.docker.com/compose/environment-variables/set-environment-variables/

Reply:
We should not use `ENTRYPOINT` to set environment variables.